internal/versions/versions_test.go

```diff
@@ -578,18 +578,55 @@ var _ = Describe("list versions", func() {
 
 		Context("for multi-arch release image", func() {
 			It("gets successfuly image using generic multiarch query", func() {
-				releaseImage, err = h.GetReleaseImage("4.11.1", common.MultiCPUArchitecture)
+				releaseImage, err = h.GetReleaseImage("4.11.1", common.MultiCPUArchitecture, true)
 				Expect(err).ShouldNot(HaveOccurred())
 				Expect(*releaseImage.OpenshiftVersion).Should(Equal("4.11.1"))
 				Expect(*releaseImage.Version).Should(Equal("4.11.1-multi"))
 			})
 			It("gets successfuly image using sub-architecture", func() {
-				releaseImage, err = h.GetReleaseImage("4.11.1", common.PowerCPUArchitecture)
+				releaseImage, err = h.GetReleaseImage("4.11.1", common.PowerCPUArchitecture, true)
 				Expect(err).ShouldNot(HaveOccurred())
 				Expect(*releaseImage.OpenshiftVersion).Should(Equal("4.11.1"))
 				Expect(*releaseImage.Version).Should(Equal("4.11.1-multi"))
 			})
 		})
+
+		Context("using weak match", func() {
```

Comment:
Can you include a case where we have a release that would weak match, but we specify strong match and we expect it to not find the release? Or is this covered with one of the existing tests in the other contexts?